examples/integration_test.py

```diff
@@ -353,6 +355,78 @@ def my_on_revoke (consumer, partitions):
     c.close()
 
 
+def verify_stats_cb():
+    """ Verify stats_cb """
+
+    def stats_cb(stats_json_str):
+        stats_json = json.loads(stats_json_str)
+        if 'test' in stats_json['topics']:
+            print("# app_offset stats for topic test partition 0: %d" % stats_json['topics']['test']['partitions']['0']['app_offset'])
```

Comment:
Maybe put this in a `try` block to give a meaningful error message if those dict keys are not available